.github/workflows/test_verification.yml

```diff
@@ -0,0 +1,144 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  Linux_gcc:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+
+        gccver: [ 7, 8, 9, 10 ]
+    steps:
+      - name: Set up cmake
+        uses: jwlawson/actions-setup-cmake@v1.11
+        with:
+          cmake-version: '3.11.x'
+
+      - name: Set up gcc
+        uses: egor-tensin/setup-gcc@v1
+        with:
+          version: ${{ matrix.gccver }}
+          platform: x64
+
+      - uses: actions/checkout@v2
+
+      - name: Make build folder
+        working-directory: ${{github.workspace}}
+        run: mkdir build
+
+      - name: CMake
+        working-directory: ${{github.workspace}}/build
+        run: cmake ../
+
+      - name: Build
+        working-directory: ${{github.workspace}}/build
+        run: cmake --build . -- -j4
+
+      - name: Test
+        working-directory: ${{github.workspace}}/build/tests/
+        run: ./tileson_tests --order rand
+
+  Linux_clang:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        clangver: [ 8, 11 ]
```

Comment:
note: clang++ version 12 and 13 are explicitly not added. With this version of clang it seems that there are std::bad_alloc exceptions raised from multiple test cases, including the benchmark test and the "Parse a whole COMPRESSED map by file" test case in `tests_main.cpp`.

* A local clang++-10 version works fine. (unfortunately I cannot upgrade to a newer clang locally)
* The issue seems to be consistently appearing over multiple runs on github actions.
* The issue is also on osx, which uses clang-13 (see comment below).
* Splitting up the benchmark test case to reduce memory consumption did not solve the issue. 
* The bad alloc seems to originate somewhere in the memory LZMA code, as I could not reproduce it locally, debugging was not possible.

Reply:
Thanks!
The `std::bad_alloc` exceptions sounds strange, but if they are not produced by `Tileson` itself I wouldn't worry too much about it. If you found a specific cause to this, I would be interested in adding a fix in the `PocketLzma` library, though.